README.md

```diff
@@ -194,27 +85,31 @@ import (
 	ofctx "github.com/OpenFunction/functions-framework-go/openfunction-context"
 )
 
-func BindingsOutput(ctx *ofctx.OpenFunctionContext, in []byte) int {
+func BindingsOutput(ctx *ofctx.OpenFunctionContext, in []byte) ofctx.RetValue {
 	log.Printf("receive greeting: %s", string(in))
-	_ := ctx.SendTo(in, "another-target")
+	_ := ctx.Send("another-target", in)
 	return 200
 }
 ```
 
-### Uninstall OpenFunction
-
-- Uninstall the latest stable version
+You can also use `ofn` to make a quick demo:
 
 ```shell
-kubectl delete -f https://raw.githubusercontent.com/OpenFunction/OpenFunction/release-0.4/config/bundle.yaml
+ofn demo
```

Comment:
add a comment before ofn demo:
// By default the demo environment will be deleted when the demo finishes.
// You can keep the demo kind cluster for further exploration by `ofn demo --auto-prune=false`
// The demo kind cluster can be deleted by xxx